app/connectors/httpParsers/ReviewHttpParser.scala

```diff
@@ -36,16 +36,17 @@ object ReviewHttpParser extends HttpParser {
           Left(MalformedResponseError)
       }
     case (_, _, response) if response.status == NOT_FOUND =>
```

Comment:
Should we repeat this match without the NOT FOUND guard? Otherwise all non NOT FOUND errors will lead to internal server error being returned.

Reply:
The new code I'm working on in the Event branch will check for all known error codes that can be returned, so will catch this